package.json

```diff
@@ -5,7 +5,7 @@
   "description": "A template for kick starting a Cloudflare Workers project",
   "main": "index.js",
   "scripts": {
-    "deploy": "wrangler deploy",
+    "deploy": "wrangler deploy --env dev",
```

Comment:
Why dev env?

Reply:
If environment is not specified it will deploy to default environment. Since we have environment `production` defined, I thought it might be good to be explicit here. I can roll it back if you want.

Reply:
But why would we deploy to the dev environment? Shouldn't we deploy to production? I would expect `nom run deploy` to do that. If you specifically want to deploy to dev, what about renaming the script to `"deploy:dev"`?

Reply:
I found it useful to deploy during the development as many things like cache are not available out of the box. I do agree that the name might be a little off -- I'm renaming the script to `deploy:dev` and `deploy:prod` to `deploy`.